src/page-response-impact/ResponseImpactDashboard.tsx

```diff
@@ -140,37 +219,112 @@ const ResponseImpactDashboard: React.FC = () => {
         <ValidDataWrapper facilities={facilities.data}>
           <PageContainer>
             <Column>
-              <ScenarioName>{scenario?.name}</ScenarioName>
-              <PageHeader>COVID-19 Response Impact as of [DATE]</PageHeader>
-              <SectionHeader>Safety of Overall Population</SectionHeader>
+              <Link to="/">
+                <BackDiv>
+                  <IconBack alt="back" src={iconBackSrc} />
+                  Back to model
+                </BackDiv>
+              </Link>
+              <PageHeader>
+                {facilities.data[0].systemType === "State Prison" ||
+                facilities.data[0].systemType === "State Jail"
+                  ? modelInputs[0].stateCode
+                  : ""}
+                {facilities.data[0].systemType === "County Jail"
+                  ? modelInputs[0].countyName
+                  : ""}{" "}
```

Comment:
@jovergaag ready for another look!

✅ combined the ternary into one.

✅ also, synced with serena:

accounting for the edge case on the /impact route is fine. 
* if county jail and county selection = total, it will display the state name

![excalibur](https://user-images.githubusercontent.com/1372946/81357877-b6cc8f00-9089-11ea-97e8-8f95735d15c4.gif)